lib/rworkflow/sidekiq_helper.rb

```diff
@@ -3,35 +3,14 @@
 module Rworkflow
   module SidekiqHelper
     def self.included(klass)
-      klass.send :extend,  ClassMethods
-      klass.send :include, InstanceMethods
+      klass.send :extend, ClassMethods
```

Comment:
I'd say let's not make it more dependent on rails until we need to make it dependent on rails